contrib/mlx_regex_setup_hugepages.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash -e
+# Dependency on package:
+#   libhugetlbfs-utils @ CentOS
+#   hugepages @ Ubuntu
+
+# Amount of hugepage memory needed by mlx-regex daemon
+min_hugemem=${MIN_HUGEMEM:-258M}
+
+# Have any hugepages been configured yet
+hugetlb=$(grep Hugetlb /proc/meminfo | awk '{ print $2 }')
```

Comment:
Can we move this to after the case statement

Reply:
Moved now Robert